src/popup/locales/en.json

```diff
@@ -236,7 +248,8 @@
       "successTitle": "Transaction successfully submitted",
       "recipientLabel": "Recipient",
       "recipientPlaceholder": "Enter address, .chain name or URL",
-      "next": "Next"
+      "next": "Next",
+      "payload": "PAYLOAD"
```

Comment:
I suggest not to use capital letters in the translation files. Instead CSS in my opinion would be a better option.